tile-group.lisp

```diff
@@ -1347,12 +1352,12 @@ direction. The following are valid directions:
     (let (shortest)
       (destructuring-bind (win-x . win-y) (window-centroid win)
         (loop :for frame :in (group-frames group)
-	      :for (frame-x . frame-y) := (frame-centroid frame)
-	      :for distance := (sqrt (+ (square (- win-x frame-x))
-					(square (- win-y frame-y))))
-	      :unless shortest
+              :for (frame-x . frame-y) := (frame-centroid frame)
+              :for distance := (sqrt (+ (square (- win-x frame-x))
+                                        (square (- win-y frame-y))))
+              :unless shortest
                 :do (setf shortest (cons distance frame))
-	      :when (> (car shortest) distance)
+              :when (> (car shortest) distance)
```

Comment:
Don't untabify code unrelated to the PR please:pray:

Reply:
Apologies! after having some issues with emacs and tabs I set up emacs to select the whole file and untabify it. I'll remove that from my .emacsrc and do it manually from now on, and fix this up.

Reply:
>  I set up emacs to select the whole file and untabify it

Samesies. I have `(untabify (point-min) (point-max))` as a before-save hook. No need to remove that setting. Just don't stage the changes when committing. If using the CLI use `git add -p`, if you use magit you can select the region of the diff before pressing s. That way you only stage those changes.
I have `()`. You can just not stage those changes when making the commit.

Reply:
ah, i didnt know of that functionality, thanks!